prow/cmd/tide/README.md

```diff
@@ -24,7 +24,96 @@ them when they have up-to-date passing test results ("tide goes out").
 - Scales efficiently so that a single instance with a single bot token can provide merge automation to dozens of orgs and repos with unique merge criteria. Every distinct 'org/repo:branch' combination defines a disjoint merge pool so that merges only affect other PRs in the same branch.
 - Provides configurable merge modes ('merge', 'squash', or 'rebase').
 
-
 ## History
 
 Tide was created in 2017 by @spxtr to replace `mungegithub`'s Submit Queue.  It was designed to manage a large number of repositories across organizations without using many API rate limit tokens by identifying mergeable PRs with GitHub search queries fulfilled by GitHub's v4 GraphQL API.
+
+## Flowchart
+
+```mermaid
+    graph TD;
+        subgraph github[GitHub]
+            subgraph org/repo/branch
+                head-ref[HEAD ref];
+                pullrequest[Pull Request];
+                status-context[Status Context];
+            end
+        end
+
+        subgraph prow-cluster
+            prowjobs[Prowjobs];
+            config.yaml;
+        end
+
+        subgraph tide-workflow
+            Tide;
+            pools;
+            filtered-pools
+            subgraph sync-sub-pool
```

Comment:
Maybe rename like the funcs where applicable? E.G
```suggestion
            subgraph syncSubpool
```

Reply:
done